fetch.bs

```diff
@@ -5630,6 +5621,81 @@ if the script checks that the URL has the right hostname.
 
 
 
+<h2 id=data-urls><code>data:</code> URLs</h2>
+
+<p>For an informative description of <code>data:</code> URLs see RFC 2397. This section replaces its
```

Comment:
<p>For an informative description of <code>data:</code> URLs see RFC 2397

Probably better to have a comma after _URLs_:

_For an informative description of <code>data:</code> URLs, see RFC 2397_